packages/mui-docs/src/branding/brandingTheme.ts

```diff
@@ -256,12 +255,10 @@ export const getDesignTokens = (mode: 'light' | 'dark') =>
       },
       warning,
       gradients: {
-        lightGrayRadio:
-          'radial-gradient(50% 50% at 50% 50%, #F0F7FF 0%, rgba(240, 247, 255, 0.05) 100%)',
-        stylizedRadio:
+        radioSubtle:
           mode === 'dark'
-            ? 'linear-gradient(rgba(0 0 0 / 0.1), rgba(0 0 0 / 0.1)), linear-gradient(254.86deg, rgba(0, 58, 117, 0.18) 0%, rgba(11, 13, 14, 0.3) 49.98%, rgba(0, 76, 153, 0.21) 100.95%)'
-            : 'linear-gradient(rgba(255 255 255 / 0.3), rgba(255 255 255 / 0.3)), linear-gradient(254.86deg, rgba(194, 224, 255, 0.12) 0%, rgba(194, 224, 255, 0.12) 0%, rgba(255, 255, 255, 0.3) 49.98%, rgba(240, 247, 255, 0.3) 100.95%)',
+            ? `radial-gradient(100% 100% at 50% 0, transparent 0, ${alpha(blue[900], 0.3)} 300%)`
```

Comment:
I'd make a slight change on the gradient here, making the lighter color start in the top-left corner. This feels more natural to me (lighter tones coming from top→bottom), but no strong opinions here.

```suggestion
            ? `radial-gradient(100% 100% at 100% 100%, transparent 0, ${alpha(blue[900], 0.3)} 300%)`
```

I wonder, though, if we shouldn't use the same card styles all over the place. I think it'd make sense to have it styled like [these](https://deploy-preview-41686--material-ui.netlify.app/material-ui/getting-started/).

Reply:
I had intentionally not done the top-to-bottom light source here so it collided less with the hero section's gradient, but it doesn't look to bad either, so that's alright! Also made these featured blog posts cards specifically use a different gradient so we had a different look & feel between the hero section (linear gradient) and the cards (radial gradient).